api/v1/routes/superadmin.py

```diff
@@ -0,0 +1,36 @@
+from api.utils.success_response import success_response
+from sqlalchemy.orm import Session
+from typing import Annotated
+from api.db.database import get_db
+from api.v1.schemas.user import UserCreate,UserBase
+from fastapi import APIRouter,Depends,status,HTTPException
+from api.v1.services.user import user_service
+
+'''
+The Endpoint created is for the creation of super admins 
+'''
+
+
+superadmin  = APIRouter(
+    prefix='/superadmin',
+    tags=['Superadmin']
+)
+
+db_dependency = Annotated[Session , Depends(get_db)]
+
+
+@superadmin.post(path='/register', status_code=status.HTTP_201_CREATED)
+
+def register_admin(user : UserCreate , db : db_dependency):
+   
+    user_created = user_service.create(db=db, schema=user)
+    user_created.is_super_admin = True
+    db.commit()
```

Comment:
did you test this, attach a post man screenshot of the working route.

what if a user has been created and we want to update role to user_admin,

create a function in the service folder that takes in the user schema, perform all necessary queries there

Reply:
@Goketech  changes made boss i added  a new service that checks if that user exists and updats the user to as super user